ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+    # Main help command
+    parser = argparse.ArgumentParser(
+        description="Reformat a NCBI Virus metadata.tsv file for a pathogen build."
+    )
+    # Add first argument
+    parser.add_argument(
+        "--metadata", help="NCBI Virus metadata.tsv file.", required=True
+    )
+    parser.add_argument(
+        "--outfile",
+        help="Output file name, e.g. processed_metadata.tsv.",
+        required=True,
+    )
+
+    return parser.parse_args()
+
+
+# === Private methods
+
+
+def _set_strain_name(record):
+    """Check Isolate_s and Strain_s to find the strain name"""
+    if record["strain"] != record["accession"]:
+        strain_name = record["strain"]
+    elif record["strain"] == record["accession"] and pd.notna(record["strain_s"]):
```

Comment:
_non-blocking_

Is the `pd.notna()` intended to allow things like the empty string thru? (Maybe `record["strain_s"]` is never the empty string because of how field types are inferred?)

Reply:
So far it's behaving as expected where the left view is before, right view is after. 

<img width="802" alt="Screen Shot 2023-04-04 at 9 08 32 PM" src="https://user-images.githubusercontent.com/1077254/229978939-90fb0a39-e4cd-4fc6-8819-f063b0083dfa.png">

The first five have a more reasonable strain name in the "strain_s" column and are replaced. 
But the 6th line "AB178040" has an empty "strain_s" so the default "strain" continues to be used. 

Do let me know if you see one where this breaks though.

The reasonable strain name can show up in either "strain" or "strain_s"

<img width="484" alt="Screen Shot 2023-04-04 at 9 17 12 PM" src="https://user-images.githubusercontent.com/1077254/229979881-17dd76fd-1218-4356-9c37-0df44b9ec3ef.png">

Reply:
> (Maybe `record["strain_s"]` is never the empty string because of how field types are inferred?)

I confirmed this is the case. `pandas.read_csv()` is used to produce the `record` objects here, and `read_csv()` by default includes the empty string in `na_values` and enables `na_filter`, so empty string values become NaNs, which then in turn pass `pd.isna()`.

![image](https://user-images.githubusercontent.com/79913/230999663-ad148e34-3140-4b31-9201-c82f8e4c8e25.png)